packages/ui/src/components/_utils/responsiveObserve.ts

```diff
@@ -67,7 +69,7 @@ const responsiveObserve = {
                 })
             }
             const mql = window.matchMedia(matchMediaQuery)
-            mql.addListener(listener)
+            mql.addEventListener('change', listener)
```

Comment:
You will be unable to remove this listener. If it is used inside effects, it should be removed on unmount. To remove it you have to store handlers reference somewhere.

Reply:
it's store in `matchHandlers` below and removed in `unregister()`